splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/TernaryOperationIT.java

```diff
@@ -484,5 +481,49 @@ public void testSTRIPNegative() throws Exception {
             Assert.assertTrue("Unexpected error code: " + e.getSQLState(),  SQLState.LANG_SYNTAX_ERROR.startsWith(e.getSQLState()));
         }
     }
+
+    private void executeAndCheckSplitPart(int fieldNumber) throws SQLException {
+        int count = 0;
+        String sCell1 = null;
+        String sCell2 = null;
+        ResultSet rs;
+
+        rs = methodWatcher.executeQuery(String.format("select split_part(e, ' ', %d), e from ", fieldNumber) + tableWatcherB);
```

Comment:
Could you please rewrite:
`String.format("select split_part(e, ' ', %d), e from ", fieldNumber) + tableWatcherB`
to:
`String.format("select split_part(e, ' ', %d), e from %s", fieldNumber, tableWatcherB)`

Reply:
done